.github/workflows/workflow-cleanup.yml

```diff
@@ -0,0 +1,35 @@
+name: Cleanup old and orphaned workflow runs
+
+on: [pull_request] # TODO: remove before merge to run on default branch
+  #schedule:
+    #- cron: '0 12 * * *' # runs at 12:00 UTC everyday
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
we started moving to `amazon coretto`.  I don't actually know what the right string is here, but it has been a decent stand in for base image and has alot cleaner security patches to save us the minor update drama later on.  

do what you feel like for this one, but for the next one let's try and see how `coretto` goes as a place to start

Reply:
I will break out the amazoncorretto into a different story